### PR TITLE
UI improvements + cleaning code

### PR DESCRIPTION
**Pull request by Niktux**
UI :
- Display alerts
- Add page titles
- fix content wrapper size
- fix wrong redirection after starting/stopping a process (but filtering still needs to be fixed)

Cleaning code : 
- Sidebar has its own controller
- Extract business logic from controller (a slim one by now ;-) to a ProcessCollectionProvider service
- Add ServerCollection and ProcessCollection classes to easy template rendering

### CODE REVIEW

File: src/Controllers/Home/Controller.php [hunk 1]
```diff
@@ -3,159 +3,97 @@
...
+            $this->addInfoFlash(sprintf('Process <b>%s</b> starting on server <b>%s</b>', $processName, $serverName));
         }
-
-        if( ! array_key_exists($serverName, $this->servers))
+        catch(\Exception $e)
         {
-            $this->addErrorFlash('Error while trying to stop process.');
-
-            return $this->redirect('home');
+            $this->addErrorFlash($e->getMessage());
```
Comment (lebris) [inline src/Controllers/Home/Controller.php:78]: Functionnal message for the user + log the techincal error message

File: src/Controllers/Home/Controller.php [hunk 2]
```diff
@@ -3,159 +3,97 @@
...
-        }
-
-        $return = $this->servers[$serverName]->stopProcess($processName);
-
-        if(! $return)
-        {
-            $this->addErrorFlash('Error while trying to stop process.');
-
-            return $this->redirect('home');
+            $this->addErrorFlash($e->getMessage());
```
Comment (lebris) [inline src/Controllers/Home/Controller.php:63]: Same here, functionnal message + log message

File: views/home.twig [hunk 3]
```diff
@@ -8,13 +8,21 @@
 {% endblock %}
 
 {% block sidebar %}
-    {% include 'sidebar.twig' %}
+    {{ render(url('sidebar')) }}
+{% endblock %}
+
+{% block pageTitle %}
+    {{title}}
+    <small>{{ processes|length }} processes</small>
```
Comment (lebris) [inline views/home.twig]: `process{% if processes|length > 1 %}es{%%}`

Comment (Niktux) [inline views/home.twig]: fixed dfa2829

**Comment by lebris** [comment]
Thanks !

**Pull request merged**